src/test/kotlin/com/zepben/evolve/cim/iec61968/infiec61968/infassetinfo/TransformerConstructionKindTest.kt

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 Zeppelin Bend Pty Ltd
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package com.zepben.evolve.cim.iec61968.infiec61968.infassetinfo
+
+import com.zepben.evolve.cim.validateEnum
+import com.zepben.testutils.junit.SystemLogExtension
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.RegisterExtension
+
+internal class TransformerConstructionKindTest {
+
+    @JvmField
+    @RegisterExtension
+    var systemErr: SystemLogExtension = SystemLogExtension.SYSTEM_ERR.captureLog().muteOnSuccess()
+
+    @Test
+    internal fun validateVsPb() {
+        validateEnum(TransformerConstructionKind.values(), com.zepben.protobuf.cim.iec61968.infiec61968.infassetinfo.TransformerConstructionKind.values())
```

Comment:
suggestion: Import the protobuf version of the enum as an alias like `validateEnum(CustomerKind.values(), PBCustomerKind.values())`.